src/main/java/uk/nhs/hee/tis/trainee/forms/service/LtftService.java

```diff
@@ -98,9 +99,12 @@ public long getAdminLtftCount(Set<LifecycleState> states) {
 
     if (states == null || states.isEmpty()) {
       log.debug("No status filter provided, counting all LTFTs.");
-      return ltftFormRepository.countByContent_ProgrammeMembership_DesignatedBodyCodeIn(groups);
+      return ltftFormRepository
+          .countByStatus_Current_StateNotInAndContent_ProgrammeMembership_DesignatedBodyCodeIn(
+              Set.of(DRAFT), groups);
     }
 
+    states = states.stream().filter(s -> s != DRAFT).collect(Collectors.toSet());
```

Comment:
Would it be worth including some sort of logging if the request included DRAFT, to indicate why the output might be not what was hoped for?

Reply:
I'm not sure logging is worth it, but I'll definitely add it to the API docs